workflow/workflow.go

```diff
@@ -173,9 +174,14 @@ func (w Workflow) migrate(telemetryDisabled bool) Workflow {
 			}
 
 			if target.CodeSamples == nil {
+				// Capitalize the first letter of language
+				sdkLabel := fmt.Sprintf("%s (SDK)", strings.ToUpper(string(target.Target[0]))+target.Target[1:])
```

Comment:
Surely there's a shared utility we can use 😅  Or at least put the capitalize in a function